packages/blog/src/utils/image.ts

```diff
@@ -16,5 +16,22 @@ export function getImageUrlFromMedia({
   format = 'large',
   fallback = '/images/WElogo.png',
 }: GetImageUrlFromMediaParams) {
-  return media ? media.formats[format].url : fallback || '/images/WElogo.png'
+  try {
+    if (!media) return fallback || '/images/WElogo.png'
+
+    const preferredFormat = media.formats[format]
+    if (preferredFormat) return preferredFormat.url
+
+    //search for the following format available
+    const nextAvailableFormatKeys = Object.keys(media.formats) as FormatType[]
+    const nextAvailableFormat = nextAvailableFormatKeys.find(formatKey => media.formats[formatKey])
```

Comment:
This logic looks a little weird to me 🤔
You iterate over the keys of `media.formats` and get the first element which has `media.formats[formatKey]` defined. But since `formatKey` is actually a key from `media.formats`, this condition will always be true, am I right?